src/models/resource.js

```diff
@@ -0,0 +1,42 @@
+const { Model } = require('sequelize');
+
+module.exports = (sequelize, DataTypes) => {
+  class Resource extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      Resource.belongsTo(models.Objective, { foreignKey: 'objectiveId' });
+    }
+  }
+  Resource.init({
+    id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+    },
+    url: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    objectiveId: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    updatedAt: {
+      allowNull: false,
+      type: DataTypes.DATE,
+    },
+    createdAt: {
+      allowNull: false,
+      type: DataTypes.DATE,
+    },
```

Comment:
Sequelize adds these columns automatically, do you need to define them? 
[https://sequelize.org/v5/manual/models-definition.html](https://sequelize.org/v5/manual/models-definition.html)
> By default, Sequelize will add the attributes createdAt and updatedAt to your model so you will be able to know when the database entry went into the db and when it was updated last.

Reply:
good to know! thanks